src/info/author.rs

```diff
@@ -42,7 +42,7 @@ impl std::fmt::Display for Author {
         if let Some(email) = &self.email {
             write!(
                 f,
-                "{}% {} <{}> {}",
+                "{:3}% {} <{}> {}",
```

Comment:
```suggestion
                "{:>3}% {} <{}> {}",
```
minor, just for readability

Reply:
Great point, done!